transform/kubernetes/kubernetes.go

```diff
@@ -268,6 +271,12 @@ func (k *KubernetesTransformPlugin) getWhiteOuts(obj unstructured.Unstructured)
 		return true
 	}
 
+	if daemonSetExtensionsGK == obj.GetObjectKind().GroupVersionKind().GroupKind() ||
```

Comment:
Looking at https://kubernetes.io/docs/reference/using-api/deprecation-guide/#deployment-v116 I think we need to handle all these from 1.16...right?

Before the new discovery changes I don't think this would have been that bad, but now we do have to worry about their being duplicates as I understand it.

Reply:
Ya, makes sense. I'll add the others. I went off of the list in MTC. Somewhat surprised we never hit an issue with the others. Good catch, thank you!